apps/root/src/state/index.ts

```diff
@@ -158,16 +128,6 @@ const createStore = (web3Service: Web3Service) =>
           ],
           byUrl: getDefaultByUrl(),
           hasLoaded: false,
-          customTokens: {
-            name: 'custom-tokens',
-            logoURI: '',
-            timestamp: new Date().getTime(),
-            tokens: [],
```

Comment:
Removing this `customTokens` object throws an error when localStorage is fully empty